.gitlab/deps_build.yml

```
@@ -144,36 +144,40 @@ build_processed_btfhub_archive:
     - git clone https://github.com/aquasecurity/btfhub-archive.git
     - cd btfhub-archive
     # Flatten btfhub-archive directories & separate arm/x64 btfs into separate directories
-    - dirs=("amzn/2" "centos/7" "centos/8" "debian/9" "debian/10" "fedora/29" "fedora/30" "fedora/31" "ol/7")
+    - dirs=("amzn/2" "centos/7" "centos/8" "debian/9" "debian/10" "fedora/28" "fedora/29" "fedora/30" "fedora/31" "ol/7" "ol/8" "rhel/7" "rhel/8")
     - |
-      for dir in ${dirs[@]}; do                                         \
-        platform=${dir%%/*}                                           &&\
-        mkdir -p "btfs-amd64/${platform}" "btfs-arm64/${platform}"    &&\
-        eval "mv ${dir}/x86_64/*.btf.tar.xz btfs-amd64/${platform}/"  &&\
-        eval "mv ${dir}/arm64/*.btf.tar.xz btfs-arm64/${platform}/";    \
+      for dir in ${dirs[@]}; do
+        platform=${dir%%/*}
+        mkdir -p "btfs-amd64/${platform}" "btfs-arm64/${platform}"
+        eval "mv ${dir}/x86_64/*.btf.tar.xz btfs-amd64/${platform}/"
+        eval "mv ${dir}/arm64/*.btf.tar.xz btfs-arm64/${platform}/"
```

Just out of curiosity, why do we need `eval` here ?
Can’t `mv` be invoked directly ?

```suggestion
        mv "${dir}"/x86_64/*.btf.tar.xz btfs-amd64/"${platform}"/
        mv "${dir}"/arm64/*.btf.tar.xz btfs-arm64/"${platform}"/
```

---

It fails if you remove the `eval` with a `stat` error. I think it has something to do with the wildcard. I tried to debug it, but was unsuccessful.